refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,887 @@
+
+Systems processing Privacy Requests (or other Concepts) containing a term unknown to them, SHOULD deploy behavior they are programmed to deploy in response to the first known supercategory of such term.
+
+In that way, interoperability CAN always be expected at the level of [Terms](#terms).
+
+### Extending the vocabulary
+
+Systems MAY specify the vocabulary used to express data being exchanged. The value indicating this version of this vocabulary is `priv.1.0`.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `vocab` | 0-1 | The name of the vocabulary used to describe data. In absence of indication `priv.1.0` is assumed. |
+
+When [addition additional terms](#extending-terms) is not sufficient, and additional Concepts or Properties are needed, it is possible to extend the vocabulary.
+
+This vocabulary is extended by defining a new identifier for the new vocabulary.
+New vocabularies MAY include other vocabularies as long as their sets of concepts, properties and terms are disjoint.  
+
+## Examples
+
+A [JSON schema of the PRIV](./priv.schema.json) is provided for convenience. To illustrate a request, let us take the example of a data subject wanting to know if a an Organization (and any of its partners if applicable) have any data on them, and if so, have their contact data deleted.
+
+This request can be represented with the following json data:
+```
+{
+  "$schema":"https://blindnet.io/schemas/priv.schema.json",
+  "request-id": "8f9066c6-1c6c-42a0-9993-e88c98d0e84d",
+  "date": "2022-06-02T14:40:39+0000",
+  "data-subject":[{
+    "dsid-schema": "email-sha-256",
+    "dsid":"7cac89a56bbf998c996f33e0b2d3bad578e05f3af8d64793c0bcac46b8c260dc"
+  }],
+  "demands":[{
+    "demand-id":"496294eb-5293-47dd-aaf8-494a0cb09134",
+    "action":"TRANSPARENCY.KNOWN"
+  },{
+    "demand-id":"86bbb28a-eee6-45e6-81d6-7101de32374b",
+    "action":"DELETE",
+    "restrictions":[{
+      "data-categories":["CONTACT"]
+    }]
+  }],
+  "target":"PARTNERS"
+
+}
+
+```
+In [here](./examples.md) we provide an overview of various Privacy Requests that a Data Subject might want to formulate according to [supported legislation](#supported-legislation) and we give indications on how each of those can be modeled with the Privacy Request Interchange Vocabulary.
+
+## Design Implications for Systems Implementing PRIV
+
+### Remembering Transfers
+
+When data about Data Subjects is transmitted from one system to another, in order to be able to process the [Targets property](#targets), and in order to reply to `TRANSPARENCY.WHO` and `TRANSPARENCY.PROVENANCE` demands, Systems MUST keep track of:
+- ID of Systesm of origin and destination, in the format of URI according to [RFC3986 of IETF](https://www.rfc-editor.org/rfc/rfc3986)
+- Full Data Capture metadata objects
+- Consents (`consent-id`) associated to the data being transferred
```

Comment:
> Consents do not appear in Data Captures or Data Capture Fragments, and are themselves connected to Data Subject Identities. My understanding from PRIV is that Consents are connected to Data Subjects, not to Data Capture
> 
> True

Should Data Capture component reject a Data Capture if, for whatever possible reason (e.g., bug, negligence), the related/required consent neither exists in the System nor is provided together with that Data Capture?

Reply:
NO. Data Capture SHOULD NOT reject a Data Capture simply because of the absence of Consent (the scope of which covers that Data Capture). This is due to:
- [incremental submission of Data Captures](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-conceptualization#data-capture---fragments), i.e. the fact that partially submitted Data Capture is still a Data Capture.
- Consent not being the only Legal Base.

I.e. A System might be configured to collect partial Data Captures (under `LEGITIMATE-INTEREST`), and ask the Data Subject to give Consent only when the Data Capture is completed. 

IMHO, it should not be the responsability of Data Capture Component to reject anything. Deciding on whether there is a legal base for keeping a Data Capture in storage should be the responsability of Privacy Computation Engine (and Privacy Compiler)